utils/releaser/src/utils.rs

```diff
@@ -69,3 +70,72 @@ pub fn root() -> PathBuf {
     }
     project_root
 }
+
+pub fn determine_new_version(bump_type: Option<String>) -> Option<String> {
+    let mut current_version: Vec<i32> = core_version()
+        .split('.')
+        .map(|f| f.parse().unwrap())
+        .collect();
+
+    let bump_type = bump_type.unwrap_or_else(|| "patch".to_string());
+    match bump_type.as_ref() {
+        "major" => current_version[0] += 1,
+        "minor" => current_version[1] += 1,
+        "patch" => current_version[2] += 1,
+        _ => panic!(
+            "{} is an undefined version bump. accepted values are: major, minor, patch",
+            bump_type
+        ),
+    }
+
+    let new_version = current_version
+        .iter()
+        .map(|f| f.to_string())
+        .collect::<Vec<String>>()
+        .join(".");
+
+    Some(new_version)
+}
+
+pub fn edit_cargo_version(cargo_path: &str, version: &str) {
+    let mut server = fs::read_to_string([cargo_path, "Cargo.toml"].join(""))
+        .unwrap()
+        .parse::<Document>()
+        .unwrap();
+
+    server["package"]["version"] = value(version);
+
+    fs::write(cargo_path, server.to_string()).unwrap();
+}
+
+pub fn bump_versions(bump_type: Option<String>) {
+    let new_version = determine_new_version(bump_type).unwrap_or_else(core_version);
+    let new_version = new_version.as_str();
+
+    let cargos_to_update = vec![
+        "/clients/admin/",
+        "/clients/cli/",
+        "/clients/egui/",
+        "/server/server/",
+        "/libs/core/",
```

Comment:
there are a handful more hissing here that now we can just keep in sync. If we own the cargo toml it should prob stay in sync.

Reply:
These are the remaining rust packages, all of which have a version of `0.1.0`. Should they all be in sync with the rest?
- `lockbook-shared`
- `test_utils`
- `egui_editor`
- `lockbook-dev`
- `releaser`
- `winstaller`

Reply:
yeah